tests/e2e/test_rosacli_cluster.go

```diff
@@ -0,0 +1,307 @@
+package e2e
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	labels "github.com/openshift/rosa/tests/ci/labels"
+	config "github.com/openshift/rosa/tests/utils/config"
+	rosacli "github.com/openshift/rosa/tests/utils/exec/rosacli"
```

Comment:
If the alias is as same as the package, I do not think we need to define the alias.

Reply:
yeah I just copied the code from the other repo. Will make a pass to harmonize that. Good point

Reply:
updated